frontend/add-client/services.component.tsx

```diff
@@ -24,177 +36,21 @@ export default function Services(props: StepComponentProps) {
-            </label>
-            <label>
-              <input
-                type="checkbox"
-                name="services"
-                value="nutrition"
-                checked={services.nutrition}
-                onChange={handleChange}
-              />
-              <span>Nutrition</span>
-            </label>
-            <label>
-              <input
-                type="checkbox"
-                name="services"
-                value="groceryTour"
-                checked={services.groceryTour}
-                onChange={handleChange}
-              />
-              <span>Grocery store tour</span>
-            </label>
-            <label>
-              <input
-                type="checkbox"
-                name="services"
-                value="cookingClass"
-                checked={services.cookingClass}
-                onChange={handleChange}
-              />
-              <span>Cooking classes</span>
-            </label>
-            <label>
-              <input
-                type="checkbox"
-                name="services"
-                value="PrEPClinic"
-                checked={services.PrEPClinic}
-                onChange={handleChange}
-              />
-              <span>PrEP clinic referral</span>
-            </label>
-            <label>
-              <input
-                type="checkbox"
-                name="services"
-                value="communityEngagement"
-                checked={services.communityEngagement}
-                onChange={handleChange}
-              />
-              <span>Community engagement and organizing</span>
-            </label>
-            <label>
-              <input
-                type="checkbox"
-                name="services"
-                value="financialCoaching"
-                checked={services.financialCoaching}
-                onChange={handleChange}
-              />
-              <span>Financial coaching</span>
-            </label>
-            <label>
-              <input
-                type="checkbox"
-                name="services"
-                value="itinRenewal"
-                checked={services.itinRenewal}
-                onChange={handleChange}
-              />
-              <span>ITIN Renewal</span>
-            </label>
-            <label>
-              <input
-                type="checkbox"
-                name="services"
-                value="VITA"
-                checked={services.VITA}
-                onChange={handleChange}
-              />
-              <span>VITA Tax help</span>
-            </label>
-            <label>
-              <input
-                type="checkbox"
-                name="services"
-                value="voterRegistration"
-                checked={services.voterRegistration}
-                onChange={handleChange}
-              />
-              <span>Voter registration</span>
-            </label>
+            {services.map(service => (
```

Comment:
Instead of hard coding the services in the frontend, we're now fetching them from the `GET /api/services` endpoint

Reply:
This is cool..